### PR TITLE
Implement DVS flow sample

### PR DESCRIPTION
**Pull request by avlaev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ This flow is used to login into the mobile app itself.
 ##### Webside login - located in folder [WebsiteLoginSample](WebsiteLoginSample/README.md)
 This flow is used to log into another app using the mobile app(the oidc flow).
 
+##### DVSSample - located in folder [DVSSample](DVSSample/README.md)
+This flow is used to sign documents.
+
 All samples use [MpinSdk.framework](https://github.com/miracl/mfa-client-sdk-ios) which is located in the root directory of the checked out repository.
 
 Instructions on how to build and run each samples can be found in the README located in the folder for each sample.
```
Comment (tatiNo5) [inline README.md]: each sample .... singular

File: README.md [hunk 2]
```diff
@@ -7,6 +7,9 @@ This flow is used to login into the mobile app itself.
 ##### Webside login - located in folder [WebsiteLoginSample](WebsiteLoginSample/README.md)
 This flow is used to log into another app using the mobile app(the oidc flow).
```
Comment (tatiNo5) [inline README.md]: Can you leave a space before the brackets?

**Comment by Foriger** [comment]
For me, the code is acceptable now :) 👍

**Comment by tatiNo5** [comment]
@avlaev, can you remove the readme changes from this PR as they are going to be done in the next PR

**Comment by avlaev** [comment]
The README.md and the images have been removed.

**Pull request merged**